wayland-backend/src/client_api.rs

```diff
@@ -286,6 +288,24 @@ impl Backend {
     pub fn dispatch_inner_queue(&self) -> Result<usize, WaylandError> {
         self.backend.dispatch_inner_queue()
     }
+
+    /// Take over handling for a proxy created by a third party.
+    ///
+    /// Returns `None` on the Rust backend.
+    ///
+    /// # Safety
+    ///
+    /// There must never be more than one party managing an object. This is only
+    /// safe to call when a third party is transferring ownership of the proxy.
+    #[inline]
+    pub unsafe fn manage_object(
```

Comment:
Is there any way to provide functions only for the system backend? Seems like there should be, but I didn't see anything related to that.

Reply:
https://github.com/Smithay/wayland-rs/blob/da5fde826432361f0d9bf996120fa2984b91e22f/wayland-backend/src/sys/mod.rs#L64-L93 is where the sys backend only methods for `client::Backend` are currently defined.